filament/src/FrameInfo.cpp

```diff
@@ -58,5 +70,51 @@ void FrameInfoManager::endFrame() {
     mIndex = (mIndex + 1) % POOL_COUNT;
 }
 
+void FrameInfoManager::update(FrameInfoManager::duration lastFrameTime) {
+    // fixme: this should be a setting
+    const float kTargetFrameTimeMilli = 16.667f;
```

Comment:
We have that info in the dynamic resolution options don't we?

Reply:
yes, but DynamicResolutionOptions will be split in two. The target framerate will go to to Renderer (basically all these will).  Right now it's hardcoded here because the output of this computation is not used yet.